### PR TITLE
Fixed tab's links

### PR DESCRIPTION
**Pull request by ar4s**

### CODE REVIEW

File: src/ralph/admin/templatetags/ralph_tags.py [hunk 1]
```diff
@@ -6,8 +6,9 @@
 
 
 @register.inclusion_tag('ralph_admin/templatetags/tabs.html')
-def views_tabs(views, name=None):
+def views_tabs(views, name=None, obj=None):
     """
     Render extra views as tabs.
     """
-    return {'views': views, 'name': name}
+    print(obj)
```
Comment (ghost) [inline src/ralph/admin/templatetags/ralph_tags.py]: rm, pls

File: src/ralph/admin/tests/tests_functional.py [hunk 2]
```diff
@@ -29,6 +23,18 @@ class ExtraDetailView(RalphDetailView, View):
     url_name = 'extra_detail_view'
 
 
+def new_test_view(klass, view_name=None):
+    view_name = view_name or 'test_view_{}'.format(new_test_view.counter)
+    class TestView(klass):
+        label = 'Test view {}'.format(new_test_view.counter)
+        url_name = view_name
+        name = view_name
```
Comment (ghost) [inline src/ralph/admin/tests/tests_functional.py]: `name` is unused?

File: src/ralph/admin/tests/tests_functional.py [hunk 3]
```diff
@@ -138,58 +145,57 @@ def test_extra_change_view_via_decorator(self):
...
 
-    def test_register_to_wrong_view(self):
-        """
-        Register detail view to list and list view to change.
-        """
-        with self.assertRaises(WrongViewClassError):
-            @register_extra_view(Foo, register_extra_view.LIST)
-            class TestView(RalphDetailView):
-                pass
+    def test_many_tabs_link(self):
```
Comment (ghost) [inline src/ralph/admin/tests/tests_functional.py:183]: I'd see docstring here. ;)

**Comment by szok** [comment]
:ok:

**Comment by mkurek** [comment]
:ok:

**Pull request merged**